src/mono/wasm/build/WasmApp.Native.targets

```
@@ -337,7 +338,9 @@
 
       <_EmccLinkStepArgs Include="@(_EmccLDFlags)" />
       <_EmccLinkStepArgs Include="--js-library &quot;%(_DotnetJSSrcFile.Identity)&quot;" />
+      <_EmccLinkStepArgs Include="--pre-js &quot;%(_DotnetJSPreFile.Identity)&quot;" />
       <_WasmLinkDependencies Include="@(_DotnetJSSrcFile)" />
+      <_WasmLinkDependencies Include="@(_DotnetJSPreFile)" />
```

And these can be removed:

```suggestion
      <_WasmLinkDependencies Include="@(_DotnetJSSrcFile)" />
```